presto-main/src/main/java/com/facebook/presto/metadata/BuiltInTypeAndFunctionNamespaceManager.java

```diff
@@ -1253,6 +1261,41 @@ public FunctionMap(FunctionMap map, Iterable<? extends SqlFunction> functions)
         }
     }
 
+    private static class ExactTypeSignature
```

Comment:
Same as the question above. Would be good to illustrate the rationale behind this together with other `equalsXXX` functions